xrootd/authorization.go

```diff
@@ -148,7 +148,7 @@ func EmitAuthfile(server server_utils.XRootDServer) error {
 		words := strings.Fields(lineContents)
 		// There exists a public access already in the authfile
 		if len(words) >= 2 && words[0] == "u" && words[1] == "*" {
-			if server.GetServerType().IsSet(config.OriginType) {
+			if server.GetServerType().IsEnabled(config.OriginType) {
```

Comment:
Again. I'm concerned about the "pelican serve" command and running multiple servers from the same process. Mostly marking it again so as to have a record of all the locations it could be an issue.

Reply:
In _think_ this part shouldn't be any issue because `GetServerType()` only returns the implementation of the `server` instance, which will only return one server type ([ref](https://github.com/PelicanPlatform/pelican/blob/b27a6edfb97357b2db1a109bd78dd2a89f2fcb8a/origin_ui/advertise.go#L39)). And the logic in `IsEnabled` is robust to tell if a single server is enabled.

Reply:
Ah, okay. Then this is fine, but I think the issues in config/config.go still apply.